### PR TITLE
More bits for Ubuntu, plus BeeGFS 7.1 and RDMA

### PR DESCRIPTION
**Pull request by oneswig**

### CODE REVIEW

File: tasks/create.yml [hunk 1]
```diff
@@ -51,20 +59,25 @@
 
 - name: Stop and delete server and client service
   block:
+  - name: Probe installed services
```
Comment (brtknr) [inline tasks/create.yml]: What’s this?

File: tasks/create.yml [hunk 2]
```diff
@@ -51,20 +59,25 @@
...
+    service_facts:
   - name: Ensures BeeGFS client service is not running if force_format is requested
     service:
       name: beegfs-client
       state: stopped
     become: yes
     when:
+    - "'beegfs-client.service' in ansible_facts.services"
```
Comment (brtknr) [inline tasks/create.yml]: Seems unnecessary to add this

File: tasks/create.yml (hunk 1 above)
Comment (oneswig) [reply tasks/create.yml]: The issue I encountered was that when we have the force flag set, but BeeGFS packages are not yet installed, the systemd service shutdown will fail.  Here I added a test to check for the presence of the service (which should be portable for modern distros), and it is conditionally disabled only if the service exists in the first place.

An alternative would be to allow the service shutdown task to fail, but I think has drawbacks.

File: tasks/create.yml [hunk 3]
```diff
@@ -51,20 +59,25 @@
...
     - beegfs_enable.oss | bool
     - beegfs_enable.client | bool
   - name: Stop BeeGFS server services
     service:
       name: "{{ item.name }}"
       state: stopped
     become: yes
-    when: item.enable | bool
+    when:
+    #- item.name in ansible_facts.services
```
Comment (oneswig) [inline tasks/create.yml]: This one needs fixing because the systemd server units have some uniqueness from (in this case) port number in the service name.

Comment (brtknr) [reply tasks/create.yml]: Should this be uncommented?

**Comment by brtknr** [comment]
LGTM

**Comment by brtknr** [comment]
Tested on CentOS ans passing.

**Pull request merged**